sdk/src/asset_handlers/png_io.rs

```diff
@@ -167,6 +190,92 @@ impl CAILoader for PngIO {
             }
         }
         None
+        */
+
+        const ITXT_CHUNK: [u8; 4] = *b"iTXt";
+
+        let ps = get_png_chunk_positions(asset_reader).ok()?;
+        let mut xmp_str: Option<String> = None;
+
+        ps.into_iter().find(|pcp| {
+            if pcp.name == ITXT_CHUNK {
+                // seek to start of chunk
+                if asset_reader.seek(SeekFrom::Start(pcp.start + 8)).is_err() {
+                    // move +8 to get past header
+                    return false;
+                }
+
+                // parse the iTxt block
+                if let Ok(key) = read_string(asset_reader, pcp.length) {
+                    if key.is_empty() || key.len() > 79 {
+                        return false;
+                    }
+
+                    // is this an XMP key
+                    if key != XMP_KEY {
+                        return false;
+                    }
+
+                    // parse rest of iTxt to get the xmp value
+                    let compressed = match asset_reader.read_u8() {
+                        Ok(c) => c != 0,
+                        Err(_) => return false,
+                    };
+
+                    let _compression_method = match asset_reader.read_u8() {
+                        Ok(c) => c != 0,
+                        Err(_) => return false,
+                    };
+
+                    let _langtag = match read_string(asset_reader, pcp.length) {
+                        Ok(s) => s,
+                        Err(_) => return false,
+                    };
+
+                    let _transkey = match read_string(asset_reader, pcp.length) {
+                        Ok(s) => s,
+                        Err(_) => return false,
+                    };
+
+                    // read iTxt data
+                    let mut data = vec![
+                        0u8;
+                        pcp.length as usize
+                            - (key.len() + _langtag.len() + _transkey.len() + 5)
+                                as usize
+                    ]; // data len - size of key - size of land - size of transkey - 3 "0" string terminators - compressed u8 - compression method u8
+                    if asset_reader.read_exact(&mut data).is_err() {
+                        return false;
+                    }
+
+                    // convert to string, decompress if needed
+                    let val = if compressed {
```

Comment:
I read "Standard XMP is written as uncompressed iTXt"  here: https://exiftool.org/TagNames/PNG.html#TextualData  So maybe we will never encounter this case, but maybe we will. It also entirely likely that we will at some point generalize this for reading exif and other things.

Reply:
I disabled it.  We can resurrect if needed.